src/web/js-interop.md

```diff
@@ -60,7 +60,7 @@ expected to work across compilers for JS interop.
 * Dart SDK constraint: `>= 2.19`
 * [`package:js`][] constraint: `>= 0.6.6`
 
-[`dart2wasm`]: https://github.com/dart-lang/sdk/blob/main/pkg/dart2wasm/dart2wasm.md#running-dart2wasm
+[`dart2wasm`]: https://github.com/dart-lang/sdk/blob/main/pkg/dart2wasm/README.md#running-dart2wasm
```

Comment:
We can also just link to the directory the `README.md` file is in, which may help provide some extra context to the user.

```suggestion
[`dart2wasm`]: https://github.com/dart-lang/sdk/blob/main/pkg/dart2wasm#running-dart2wasm
```